src/constants.mjs

```diff
@@ -125,17 +125,45 @@ export const DOC_SLUG_ENVIRONMENT = 'environment-variables-1';
 // JavaScript globals types within the MDN JavaScript docs
 // @see DOC_MDN_BASE_URL_JS_GLOBALS
 export const DOC_TYPES_MAPPING_GLOBALS = {
-  ...Object.fromEntries([
-    'AggregateError', 'Array', 'ArrayBuffer', 'DataView', 'Date', 'Error',
-    'EvalError', 'Function', 'Map', 'NaN', 'Object', 'Promise', 'Proxy', 'RangeError',
-    'ReferenceError', 'RegExp', 'Set', 'SharedArrayBuffer', 'SyntaxError', 'Symbol',
-    'TypeError', 'URIError', 'WeakMap', 'WeakSet',
-
-    'TypedArray',
-    'Float32Array', 'Float64Array',
-    'Int8Array', 'Int16Array', 'Int32Array',
-    'Uint8Array', 'Uint8ClampedArray', 'Uint16Array', 'Uint32Array',
-  ].map(e => [e, e])),
+  ...Object.fromEntries(
```

Comment:
why theses changes ?

Reply:
Just prettier's auto formatting changes

Reply:
oh sad

Reply:
Is this the project's prettier settings or yours?

Reply:
Project
<img width="363" alt="image" src="https://github.com/user-attachments/assets/a807a82b-7648-491a-9e5f-90a5c619dd7f" />